src/Features/CSharp/Portable/CodeLens/CSharpCodeLensMemberFinder.cs

```diff
@@ -83,5 +84,41 @@ public override void VisitRecordDeclaration(RecordDeclarationSyntax node)
             _memberBuilder.Add(new CodeLensMember(node, node.Identifier.Span));
             base.VisitRecordDeclaration(node);
         }
+
+        public override void VisitDelegateDeclaration(DelegateDeclarationSyntax node)
+        {
+            _memberBuilder.Add(new CodeLensMember(node, node.Identifier.Span));
+        }
+
+        public override void VisitEnumMemberDeclaration(EnumMemberDeclarationSyntax node)
+        {
+            _memberBuilder.Add(new CodeLensMember(node, node.Identifier.Span));
+        }
+
+        public override void VisitFieldDeclaration(FieldDeclarationSyntax node)
+        {
+            foreach (var variable in node.Declaration.Variables)
+            {
+                _memberBuilder.Add(new CodeLensMember(variable, variable.Identifier.Span));
+            }
+        }
+
+        public override void VisitEventFieldDeclaration(EventFieldDeclarationSyntax node)
+        {
+            foreach (var variable in node.Declaration.Variables)
+            {
+                _memberBuilder.Add(new CodeLensMember(variable, variable.Identifier.Span));
+            }
+        }
+
+        public override void VisitEventDeclaration(EventDeclarationSyntax node)
+        {
+            _memberBuilder.Add(new CodeLensMember(node, node.Identifier.Span));
+        }
+
+        public override void VisitDestructorDeclaration(DestructorDeclarationSyntax node)
```

Comment:
note: won't ever have results.  so this may be likely to annoy people as it will always have "0 references" showing for it.  Would prefer to not do this.

Reply:
I think that's fine, for a few reasons:

1. Behavior matches current Visual Studio behavior
2. Destructors are rare and generally discouraged, so most users won't ever see the difference
3. For the remaining users, it's less likely someone will ask why it shows 0 than for someone to ask why it's missing